main.pm

```diff
@@ -401,12 +401,17 @@ sub load_inst_tests() {
         loadtest "installation/proxy_start_2nd_stage.pm";
     }
     loadtest "installation/sle11_wait_for_2nd_stage.pm";
-    if (noupdatestep_is_applicable && check_var('FLAVOR', 'Server-DVD')) {
+    if (noupdatestep_is_applicable && check_var('FLAVOR', 'Server-DVD') || check_var('FLAVOR', 'Server-MINI')) {
         loadtest "installation/user_settings_root.pm";
     }
     loadtest "installation/sle11_network.pm";
-    loadtest "installation/sle11_ncc.pm";
-    if (noupdatestep_is_applicable && check_var('FLAVOR', 'Server-DVD')) {
+    if (get_var('NCC')) {
+        loadtest "installation/sle11_ncc.pm";
+        loadtest "installation/sle11_online_update.pm";
+    } else {
+        loadtest "installation/sle11_skip_ncc.pm";
+    }
+    if (noupdatestep_is_applicable && check_var('FLAVOR', 'Server-DVD') || check_var('FLAVOR', 'Server-MINI')) {
```

Comment:
Please put () brackets around the two check_vars to make sure the &&'s and ||'s are evaluated correctly - we want to make sure that "(noupdatestep_is_applicable && (check_var('FLAVOR', 'Server-DVD') || check_var('FLAVOR', 'Server-MINI')))"